src/rules/no-deprecated-classes.js

```diff
@@ -56,12 +56,13 @@ module.exports = {
   },
 
   create (context) {
-    return context.parserServices.defineTemplateBodyVisitor({
+    const sourceCode = context.getSourceCode()
```

Comment:
`getSourceCode()` is depricated. Documentation suggests to use `context.sourceCode`.
https://eslint.org/docs/latest/extend/custom-rules#the-context-object

Reply:
Ok, I've replaced `context.getSourceCode()` with `context.sourceCode` in all 9 locations.
Please merge my changes. A lot of people are waiting for them.